src/dmd/root/env.d

```diff
@@ -19,6 +19,8 @@ import dmd.root.string;
 version (Windows)
     private extern (C) int putenv(const char*) nothrow;
 
+nothrow:
+
```

Comment:
?

Reply:
> I presume he means

I prefer to ask @ibuclaw to explain, as I tire of guessing :-)